Resources/Prototypes/_LostParadise/Entities/Clothing/Head/helmet.yml

```
@@ -125,4 +125,15 @@
         Slash: 0.85
-        Piercing: 0.85
\ No newline at end of file
+        Piercing: 0.85
+
+- type: entity
+  parent: LPPClothingHeadUrbanHelmetPG
+  id: LPPClothingHeadHelmetNPZAssault
+  name: NPZ assault helmet
+  description: Robust helmet with additional armor on neck, pretty good for asaults.
+  components:
+  - type: Sprite
```

yamllint

[error] wrong indentation: expected 4 but found 2

(indentation)